packages/experimental/RadioGroup/src/Radio/useRadio.ts

```diff
@@ -52,9 +52,10 @@ export const useRadio = (props: RadioProps): RadioState => {
     onFocus: changeSelection,
   });
 
-  const accessibilityActionsProp = React.useCallback(() => {
-    accessibilityActions ? [...defaultAccessibilityActions, ...accessibilityActions] : defaultAccessibilityActions;
-  }, []);
+  const accessibilityActionsProp = React.useMemo(
+    () => (accessibilityActions ? [...defaultAccessibilityActions, ...accessibilityActions] : defaultAccessibilityActions),
+    [],
```

Comment:
Should be memoized on accessibilityActions, otherwise this won't change if accessibilityActions changes

Reply:
Added! Realized this change got left out of my latest commits.